src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -36,6 +36,12 @@ const Authenticate = (() => {
         $submit_status_uns,
         $submit_table_uns;
 
+    const personal_details_map = {
+        place_of_birth: localize('Place of birth'),
+        residence     : localize('Country of residence'),
+        phone         : localize('Phone number'),
+    };
```

Comment:
this is not the full list of errors. you can find it here https://github.com/binary-com/deriv-app/blob/dev/packages/core/src/Modules/Cashier/Components/Error/error.jsx#L48-L56

perhaps we can refactor it so it can be declared once and used in both places